.drone.yml

```diff
@@ -89,13 +88,9 @@ pipeline:
       - docker-compose -f docker-compose-build.yml build --no-cache nodejs python-dev python-prod
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      # in case if "Push Hooks" is enabled in Drone settings GUI
-      - /root/prog/shopelectro/docker/.env:/drone/src/github.com/fidals/shopelectro/commit/${DRONE_COMMIT_SHA}/docker/.env
-      # in case if "Pull Request Hooks" is enabled in Drone settings GUI
-      - /root/prog/shopelectro/docker/.env:/drone/src/github.com/fidals/shopelectro/pull/${DRONE_PULL_REQUEST}/docker/.env
       # for nodejs build
       - /tmp/cache/drone/shopelectro/site-packages/${DRONE_COMMIT_SHA}/site-packages:/usr/app/deps
-      - /tmp/cache/drone/shopelectro/front_build/${DRONE_COMMIT_SHA}:/usr/app/front/build
+      - /tmp/cache/drone/shopelectro/front_build/${DRONE_COMMIT_SHA}:/drone/src/github.com/fidals/shopelectro/front/build
```

Comment:
read drone docs and now don't understand why we are needed some mount.
It should be available from container's workspace without any mounting

http://docs.drone.io/workspace/

Reply:
@duker33 I already have created an issue for it fidals/refarm-site#132

Reply:
automounting works only with named volumes, got it